SeaPublicWebsite/Models/EnergyEfficiency/AccessibleLoftSpaceViewModel.cs

```diff
@@ -10,6 +10,8 @@ public class AccessibleLoftSpaceViewModel
         public AccessibleLoftSpace? AccessibleLoftSpace { get; set; }
 
         public string Reference { get; set; }
-        public bool Change { get; set; }
+        public PageName? EntryPoint { get; set; }
+        
+        public string BackLink { get; set; }
```

Comment:
EntryPoint represents the page we selected to change answer to, from answer summary (it is intended to replace the change bool)
BackLink represents to what page the back button should point to